specs/contracts.spec.yaml

```diff
@@ -0,0 +1,77 @@
+# spec for deploying contracts
+# {{name}} is replaced with one of the following, whichever matches first
+# - the deployed contract address whose name matches `name` (not factoryName)
+# - variables of the running chain, e.g. {{chain.chainName}}, {{chain.chainId}}
+# NOTE: order of the contracts matters, as some contracts depend on others
+# contracts with no deps should be placed before those with deps
+# deployer: must be a valid name in accountRegistry; default to 'default' if not specified
+
+- name: LightClient
+  description: 'DummyLightClient'
+  factoryName: 'DummyLightClient'
+  deployer: 'KEY_POLYMER'
+
+- name: Ibc
+  description: 'IBC library'
+  factoryName: 'Ibc'
+  deployer: 'KEY_POLYMER'
+
+- name: IbcUtils
+  description: 'IBC utils library'
+  factoryName: 'IbcUtils'
+  deployer: 'KEY_POLYMER'
+
+- name: Dispatcher
+  description: 'IBC Core contract'
+  factoryName: 'Dispatcher'
+  libraries:
+    - name: 'contracts/libs/Ibc.sol:Ibc'
+      address: '{{Ibc}}'
+  deployer: 'KEY_POLYMER'
+
+- name: DispatcherProxy
+  description: 'Dispatcher proxy contract'
+  factoryName: 'ERC1967Proxy'
+  deployArgs:
+    - '{{Dispatcher}}'
+    - '$INITARGS'
```

Comment:
This comment is on me really since I didn't add the necessary docs about `$INITARGS` but can we add a line or two explaining what this actually does?

Reply:
should be done now! and I added comments for all arguments in the yaml files